### ISSUE TITLE
Question: Is Giraffe currently incompatible with .NET 5?

### ISSUE THREAD

**Issue by GavinRay97** [giraffe-fsharp/Giraffe#448]
Apologies if this is a silly question. I want to learn F# through building some apps with Giraffe, and I don't really have prior experience with .NET. So I can't tell whether there's incompatibility or if I just don't know what I'm doing.

I had updated the `.fsproj` file like this:
```fsproj
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
    <AssemblyName>fsharp-giraffe.App</AssemblyName>
    <EnableDefaultContentItems>false</EnableDefaultContentItems>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Giraffe" Version="5.0.0-alpha-003" />
    <PackageReference Include="TaskBuilder.fs" Version="2.1.*" />
  </ItemGroup>

  <ItemGroup>
    <Compile Include="Models.fs" />
    <Compile Include="HttpHandlers.fs" />
    <Compile Include="Program.fs" />
  </ItemGroup>

  <ItemGroup>
    <None Include="web.config" CopyToOutputDirectory="PreserveNewest" />
  </ItemGroup>
</Project>
```

But when trying to run it in VS Code with Ionide, I get this:
```sh
☸ microk8s in ~/Projects/fsharp-giraffe via •NET v5.0.100 on ☁️  us-east-1 took 16s 
❯ mono '/home/user/Projects/fsharp-giraffe/bin/Debug/net5.0/fsharp-giraffe.App.dll'
The entry point method could not be loaded due to Could not load file or assembly 'System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
```

I searched the project for references to "System.Runtime" and didn't seem to be able to find anything.

**Comment by baronfel** [comment]
Quick note: you wouldn't run with mono here. Try running the project with 'dotnet run' inside the folder containing the project. Or via 'dotnet path/to/projects/dll' after building.

**Comment by dustinmoris** [comment]
Hi, FYI - I've just released Giraffe `5.0.0-rc-1` which has been updated to .NET 5 (`net5.0`).
